paper/paper.md

```diff
@@ -0,0 +1,135 @@
+for the next project where one of the aforementioned choices changes, and the
+process of designing a workflow must begin again from scratch.
+
+Software packages such as Radonpy [@radonpy_2022] exist that provide an automated workflow for
+building molecules and bulk structures to calculating physical properties of polymers.
+However, these tools may not be suitable for modeling complex experimental
+processes that extend beyond measuring material properties, such as
+simulating fusion welding of polymer interfaces
+[@aggarwal_molecular_2020, @bukowski_load-bearing_2021] and surface wetting
+[@fan_wetting_1995, bamane_wetting_2021].
+
+`flowerMD` is a Python package that consolidates and automates
+end-to-end workflows for modeling such processes with a focus on organic molecules.
+Following the principals of Transparent, Reproducible, Usable by others, and Extensible (TRUE) [@TRUE_2020]
+software design, the modular design of `flowerMD` facilitates building and
+running workflows for specific materials science research applications,
+while reducing the cognitive load and programming demands on the user's part.
+
+# Building Blocks
+`flowerMD` is extensible. Modular base classes serve as building blocks that lay the
+foundation for constructing segregated workflow recipes designed for specific applications.
+The resultant recipes are agnostic to choices such as chemistry, model resolution
+(e.g. atomistic vs. coarse grained) and forcefield selection.
+This is accomplished via three base classes:
+
+• The `Molecule` class utilizes the mBuild [@mbuild_2016] and GMSO [@gmso] packages to initialize chemical
+structures from a variety of input formats. This class provides methods
+for building polymers and copolymer structures, and supports a straightforward
+coarse-graining process by leveraging SMARTS matching.
+
+• The `System` class serves as an intermediary between molecular initialization
+and simulation setup. This class builds the initial configuration and
+applies a chosen forcefield that defines particle interactions.
+
+• The `Simulation` class adds a layer on top of the HOOMD-blue simulation object,
+adding additional methods and features to simplify the process of starting and
+resuming a HOOMD-blue simulation.
+
+Additionally, `flowerMD` offers a library pre-defined subclasses of these base classes
+including common polymers, forcefields, and bulk system initialization algorithms.
+
+# Recipes
+`flowerMD` offers the following two ready-to-go recipes to illustrate how the design creates
+potential for expanding the library of open-source and version-controlled workflows. The included example
+code demonstrates how the modularity of `flowerMD` allows use and re-use of workflows, using
+ pre-built recipe subclasses included in the repository: `SlabSimulation`, `WeldSimulation`, and `Tensile`.
+This script creates two "slabs" of polyethylene, simulates welding at their interface, then simulates a tensile
```

Comment:
I think this might be presented more effectively if we have a list of the steps (kind of like following a recipe)

1. `flowerMD.modules.welding.SlabSimulation`: Creates a dense polymer melt with two flat surfaces
2. `flowerMD.modules.welding.Interface`: Builds an interface system between two slabs
3. `flowerMD.modules.welding.WeldSimulation`: Runs weld simulation resulting in a single fused system
4. `flowerMD.library.simulations.Tensile`: Performs a tensile test simulation on the output of the weld simulation.